### ISSUE TITLE
api should return empty list if directory is not in etcd

### ISSUE THREAD

**Issue by mkudlej** [Tendrl/api#112]
If there is no job, cluster, node or different entity there is no directory in Etcd.  In this case API returns:
```
$ curl -X GET -i -k -H "Authorization: Bearer $(cat token.txt)" ${URL}/jobs
HTTP/1.1 404 Not Found                                                                 
Content-Type: application/json                                                         
Access-Control-Allow-Origin: *                                                         
Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE                              
Access-Control-Allow-Headers: Origin,Content-Type,Accept,Authorization,X-Requested-With
X-Content-Type-Options: nosniff                                                        
Content-Length: 35                                                                     

{"errors":{"message":"Not found."}}
```
i think that it should return empty list:
```
HTTP/1.1 200 OK
Content-Type: application/json
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
Access-Control-Allow-Headers: Origin,Content-Type,Accept,Authorization,X-Requested-With
X-Content-Type-Options: nosniff
Content-Length: 2

[]
```

**Comment by anivargi** [comment]
The issue will be handled as part of https://github.com/Tendrl/api/pull/107 which is WIP

**Comment by anivargi** [comment]
Fixed in v1.3.0
